sdk/core/Azure.Core/src/Pipeline/Policies/TelemetryPolicy.cs

```diff
@@ -9,24 +9,46 @@ namespace Azure.Core.Pipeline.Policies
 {
     public class TelemetryPolicy : SynchronousHttpPipelinePolicy
     {
-        private readonly string _header;
+        private readonly Assembly _clientAssembly;
 
-        public TelemetryPolicy(Assembly clientAssembly, string applicationId)
+        private string _header;
+
+        private string _applicationId;
+
+        private readonly bool _disable = EnvironmentVariableToBool(Environment.GetEnvironmentVariable("AZURE_TELEMETRY_DISABLED")) ?? false;
+
+        public string ApplicationId
```

Comment:
Should this be static?

Reply:
I still want it to be settable per client even if we have a static default.